lib/experimental/fonts/font-library/README.md

```diff
@@ -0,0 +1,248 @@
+- **Uninstall font family**: Is to remove the font family from the Font Library permanently.
+
+### Different types of installations
+
+The library allows extenders to define how WordPress will install fonts. When users install a font family, its definition will always be saved to the database. What can vary is where are the font face file assets stored.
+
+Review the following different types of installations to find what works best for your use case:
+#### Install a font with no font file assets (system fonts)
+In a situation where you want to install a font family that doesn't need any font assets to work, these fonts are usually called system fonts. You can define these font families like this:
+```json
+{
+    "name": "Humanist",
+    "slug": "humanist",
+    "fontFamily": "Seravek, 'Gill Sans Nova', Ubuntu, Calibri, 'DejaVu Sans', source-sans-pro, sans-serif",
+
+}
+```
+
+#### Install a font with external assets
+Let's say you want to install a font family that needs to add font faces using external font file assets. In this way, your font face assets will always be served from the external URL you indicated in the `src` property of the font faces. Your font families should look like this:
+```json
+{
+    "name": "Piazzolla",
+    "slug": "piazzolla",
+    "fontFamily": "Piazzolla, serif",
+    "fontFace": [
+        {
+          "src": "http://fonts.gstatic.com/s/piazzolla/v33/N0b52SlTPu5rIkWIZjVKKtYtfxYqZ4RJBFzFfYUjkSDdlqZgy7LYxnLy1AHfAAy5.ttf",
+          "fontWeight": "400",
+          "fontStyle": "normal",
+          "fontFamily": "Piazzolla"
+        },
+    ]
+}
+```
+
+####  Install a font from external sources and serve it from your site
+Let's say you want to install a font family that uses assets located outside of your site and download it while installing to serve them always from your WordPress `/wp-content/fonts` folder. This can be useful if you want to avoid depending on external sites for technical or legal reasons. In that case, you need to set the `downloadFromUrl` property on each font face.
+
+```json
+{
+    "name": "Piazzolla",
+    "slug": "piazzolla",
+    "fontFamily": "Piazzolla, serif",
+    "fontFace": [
+        {
+          "downloadFromUrl": "http://fonts.gstatic.com/s/piazzolla/v33/N0b52SlTPu5rIkWIZjVKKtYtfxYqZ4RJBFzFfYUjkSDdlqZgy7LYxnLy1AHfAAy5.ttf",
+          "fontWeight": "400",
+          "fontStyle": "normal",
+          "fontFamily": "Piazzolla"
+        },
+    ]
+}
+```
+
+#### Install a font providing the font asset
+Let's say you want to install a font family using the font files you are providing in an HTTP request. These font file assets will be stored in your WordPress `/wp-content/fonts` folder and they will always be served from your site. As the previous way, this can be useful if you want to avoid depending on external sites for technical or legal reasons. You need to add the file to your HTTP request and add a reference for it in the font face definition.
+
+
```

Comment:
Same as above; this might not always be the directory where fonts are served from in the future.